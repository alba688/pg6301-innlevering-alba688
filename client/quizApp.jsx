import React from "react";

export function QuestionPage() {
  return (
    <div>
      <h1>Question Page</h1>
    </div>
  );
}

export function FrontPage() {
  return (
    <div>
      <h1>Quiz Application</h1>
    </div>
  );
}

export function AnswerPage() {
  return (
    <div>
      <h1>Answer Page</h1>
    </div>
  );
}
