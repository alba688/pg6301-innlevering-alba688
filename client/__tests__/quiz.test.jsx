import * as React from "react";
import ReactDOM from "react-dom";
import { AnswerPage, FrontPage, QuestionPage } from "../quizApp";

describe("Quiz Application", () => {
  it("Shows a question", () => {
    const element = document.createElement("div");
    ReactDOM.render(<QuestionPage />, element);
    expect(element.innerHTML).toMatchSnapshot();
  });

  it("Shows front page", () => {
    const element = document.createElement("div");
    ReactDOM.render(<FrontPage />, element);
    expect(element.innerHTML).toMatchSnapshot();
  });

  it("Shows answer page", () => {
    const element = document.createElement("div");
    ReactDOM.render(<AnswerPage />, element);
    expect(element.innerHTML).toMatchSnapshot();
  });
});
