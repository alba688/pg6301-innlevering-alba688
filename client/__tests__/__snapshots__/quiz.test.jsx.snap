// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quiz Application Shows a question 1`] = `"<div><h1>Question Page</h1></div>"`;
